workflow-trace-viewer/src/jvmMain/kotlin/com/squareup/workflow1/traceviewer/ui/WorkflowInfoPanel.kt

```diff
@@ -98,19 +104,33 @@ private fun NodePanelDetails(
     }
     item {
       Text(
-        text = "Workflow Details",
-        style = MaterialTheme.typography.h6,
-        modifier = Modifier.padding(top = 8.dp, bottom = 8.dp)
+        text = "${node.current.parent} (ID: ${node.current.parentId})",
+        style = MaterialTheme.typography.subtitle2,
+        color = Color.Gray,
+        modifier = Modifier.padding(top = 8.dp)
```

Comment:
We should extract constants/styles for these padding values etc.

(If that needs to be done separately/later, that's fine too!)